relm4/Cargo.toml

```diff
@@ -56,13 +56,14 @@ relm4-macros = { path = "../relm4-macros" }
 criterion = { version = "0.4", default-features = false }
 
 # For the examples
-tokio = { version = "1.25", features = [
+relm4-icons = { version = "0.6.0-beta.1", features = ["plus", "minus"]}
+tokio = { version = "1.26", features = [
     "rt",
     "macros",
     "time",
     "rt-multi-thread",
 ] }
-futures = "0.3.26"
```

Comment:
The dependencies that have been imported in the 'dependencies' section can be used directly, there is no need to import them again here.